x/crisis/module.go

```diff
@@ -63,15 +62,13 @@ func (AppModuleBasic) GetQueryCmd(_ *codec.Codec) *cobra.Command { return nil }
 type AppModule struct {
 	AppModuleBasic
 	keeper Keeper
-	logger log.Logger
 }
 
 // NewAppModule creates a new AppModule object
-func NewAppModule(keeper Keeper, logger log.Logger) AppModule {
```

Comment:
++

Reply:
I thought I don't need the logger there because it's present in the module's keeper. And previously at line 105, it was passed as a parameter to the keeper's method named `AssertInvariants`, which we don't require now.